{{ cookiecutter.project_name }}/{{ cookiecutter.app_name }}/tests/test_module.py

```diff
@@ -1,22 +0,0 @@
-# -*- coding: utf-8 -*-
-"""
-Testing of the module level stuff itself
-"""
-import unittest
-
-import semantic_version
-
-
-class TestModule(unittest.TestCase):
-    """
-    Test core module features, like asserting the version
-    and making sure we are exposing our classes.
-    """
-
-    @staticmethod
-    def test_version():
-        """
-        Verify we have a valid semantic version
-        """
-        import {{ cookiecutter.app_name }}
-        semantic_version.Version({{ cookiecutter.app_name }}.__version__)
```

Comment:
Yup, looks like it: https://github.com/mitodl/cookiecutter-djangoapp/blob/c9f00985e2fd7e15b477dc627b5ca1173958cf4a/%7B%7B%20cookiecutter.project_name%20%7D%7D/%7B%7B%20cookiecutter.project_name%20%7D%7D/test_settings.py#L142-L146